lib/screens/wallet/widgets/total_customer_currency.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:brasil_fields/brasil_fields.dart';
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+import '../controller/crypto_controller.dart';
+import '../model/crypto_model.dart';
+import '../provider/wallet_providers.dart';
+
+class TotalCurrencyCustomer extends HookConsumerWidget {
+  const TotalCurrencyCustomer({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    bool isVisible = ref.watch(isVisibleProvider.state).state;
```

Comment:
Nesse caso, não é necessário evidenciar ref.watch(isVisibleProvider**.state).state**, é possível escrever só ref.watch(isVisibleProvider). **.state).state** ao meu saber é só necessário quando for mudar o valor dentro do state do provider por fora.